docs/running-on-kubernetes.md

```diff
@@ -436,6 +436,13 @@ When there exists a log collection system, you can expose it at Spark Driver `Ex
 spark.ui.custom.executor.log.url='https://log-server/log?appId={{APP_ID}}&execId={{EXECUTOR_ID}}'
 ```
 
+You can add custom variables to this url template, populated with the values of existing executor environment variables like
+
+```
+spark.executorEnv.SPARK_EXECUTOR_ATTRIBUTE_YOUR_VAR='$(EXECUTOR_ENV_VAR)'
```

Comment:
`EXECUTOR_ENV_VAR` part looks confusing. According to the above description, `EXISTING_EXECUTOR_ENV_VAR` could be better.

Reply:
Sounds good, done in e0eba77420a.